docs/reference/arm.md

```diff
@@ -3,66 +3,42 @@ title: "ARM support"
-* Modeling data inside Redis can be very useful in order to make in-device decisions for appliances that must respond very quickly or when the remote servers are offline.
-* Redis can be used as a communication system between the processes running in the device.
-* The append-only file storage of Redis is well suited for SSD cards.
-* The stream data structure included in Redis versions 5.0 and higher was specifically designed for time series applications and has a very low memory overhead.
+* Valkey has a very small memory footprint and CPU requirements. It can run in small devices like the Raspberry Pi Zero without impacting the overall performance, using a small amount of memory while delivering good performance for many use cases.
+* The data structures of Valkey are often an ideal way to model IoT/embedded use cases. Some examples include accumulating time series data, receiving or queuing commands to execute or respond to send back to the remote servers, and so forth.
+* Modeling data inside Valkey can be very useful in order to make in-device decisions for appliances that must respond very quickly or when the remote servers are offline.
+* Valkey can be used as a communication system between the processes running in the device.
+* The append-only file storage of Valkey is well suited for SSD cards.
+* The stream data structure was specifically designed for time series applications and has a very low memory overhead.
 
-## Redis /proc/cpu/alignment requirements
+## Valkey /proc/cpu/alignment requirements
 
 Linux on ARM allows to trap unaligned accesses and fix them inside the kernel
 in order to continue the execution of the offending program instead of
-generating a `SIGBUS`. Redis 4.0 and greater are fixed in order to avoid any kind
-of unaligned access, so there is no need to have a specific value for this
-kernel configuration. Even when kernel alignment fixing set as disabled Redis should
+generating a `SIGBUS`. Valkey avoids any kind of unaligned access, 
+so there is no need to have a specific value for this
+kernel configuration. Even when kernel alignment fixing set as disabled Valkey should
 run as expected.
 
-## Building Redis in the Pi
+## Building Valkey in the Pi
 
-* Download Redis version 4.0 or higher.
 * Use `make` as usual to create the executable.
 
 There is nothing special in the process. The only difference is that by
-default, Redis uses the `libc` allocator instead of defaulting to `jemalloc`
+default, Valkey uses the `libc` allocator instead of defaulting to `jemalloc`
 as it does in other Linux based environments. This is because we believe
 that for the small use cases inside embedded devices, memory fragmentation
 is unlikely to be a problem. Moreover `jemalloc` on ARM may not be as tested
-as the `libc` allocator.
-
-## Performance
```

Comment:
@stockholmux Removed this section as performance test done way long back on redis 4.0, I am not sure whether they are really relevant now. Hopefully, someone will do the test and based on Valkey actual performance results PI, we can bring this section back.